pkg/kv/kvserver/replica_consistency_diff.go

```diff
@@ -36,46 +35,38 @@ type ReplicaSnapshotDiff struct {
 type ReplicaSnapshotDiffSlice []ReplicaSnapshotDiff
 
 // WriteTo writes a string representation of itself to the given writer.
-func (rsds ReplicaSnapshotDiffSlice) WriteTo(w io.Writer) (int64, error) {
-	n, err := w.Write([]byte("--- leaseholder\n+++ follower\n"))
-	if err != nil {
-		return 0, err
-	}
+func (rsds ReplicaSnapshotDiffSlice) WriteTo(buf *redact.StringBuilder) {
```

Comment:
Ugh I don't see the point of implementing `WriteTo`. 

Why don't you make this a standard-looking `SafeFormat` method instead